airbyte-integrations/connectors/destination-azure-blob-storage-python/destination_azure_blob_storage_python/azure.py

```diff
@@ -115,17 +122,6 @@ def write_parquet(self, messages, stream_name, stream_schema):
 
         buf = pa.BufferOutputStream()
 
-        #build path
-        path_name = ""
```

Comment:
the `path_name` variable still exists, but the 'build path' code has been deleted - how does the new write functionality allow for pathing, or does it remove it?

Reply:
Oh, moved the path building over to the stream writer file. Reduced a bit of redundant code and allowed me to use it for the file overwrite code as well. Path name now comes from the parameters